### ISSUE TITLE
Remove annotation processors added automatically by gradle-plugin

### ISSUE THREAD

**Issue by wetted** [micronaut-projects/micronaut-starter#2186]
See https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/872 and https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/873

ping @sdelamo I think you had some ideas about this.

**Comment by sdelamo** [comment]
> See [micronaut-projects/micronaut-gradle-plugin#872](https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/872) and [micronaut-projects/micronaut-gradle-plugin#873](https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/873)
> 
> ping @sdelamo I think you had some ideas about this.

I don't think we can do this change until a version of the Gradle plugin adds the annotation processors for every module.
